.circleci/config.yml

```diff
@@ -31,7 +36,7 @@ jobs:
           name: install python dependencies
           command: |
             python3 -m venv venv
-            . venv/bin/activate
+            . venv/bin/activate 
```

Comment:
Was this just a finger slip or is this needed?

Reply:
I probably did it to trigger a CI run, not needed